libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -184,6 +184,21 @@
         LOGIN_INSERTED_INVALID_URL,
         LOGIN_AUTOFILL_CREDENTIALS_FILLED,
         LOGIN_AUTOFILL_CREDENTIALS_UPDATED,
+        LOGIN_PROLOGUE_VIEWED,
+        LOGIN_EMAIL_FORM_VIEWED,
+        LOGIN_EMAIL_RETRY_VIEWED,
+        LOGIN_MAGIC_LINK_OPEN_FORM_VIEWED,
+        LOGIN_MAGIC_LINK_REQUEST_FORM_VIEWED,
+        LOGIN_PASSWORD_FORM_VIEWED,
+        LOGIN_PASSWORD_RETRY_VIEWED,
+        LOGIN_URL_FORM_VIEWED,
+        LOGIN_URL_RETRY_VIEWED,
+        LOGIN_URL_HELP_VIEWED,
```

Comment:
Since we now have a textual "HELP" button/action on the login screens, this event's name becomes less clear. May I suggest renaming it to something like "LOGIN_URL_EXPLANATION_VIEWED"?